messages/commands/dailydebitlimit.go

```diff
@@ -27,3 +28,24 @@ func (m ConsumeDailyDebitLimit) MessageDescription() string {
 		m.AccountID,
 	)
 }
+
+// Validate returns a non-nil error if the message is invalid.
+func (m ConsumeDailyDebitLimit) Validate() error {
+	if m.TransactionID == "" {
+		return errors.New("ConsumeDailyDebitLimit must not have an empty transaction ID")
+	}
+	if m.AccountID == "" {
+		return errors.New("ConsumeDailyDebitLimit must not have an empty account ID")
+	}
+	if err := m.DebitType.Validate(); err != nil && !m.DebitType.IsDebit() {
+		return errors.New("ConsumeDailyDebitLimit must have a valid debit type")
```

Comment:
This one throws away the original error altogether, probably better split into separate checks.

Reply:
Split debit type validation for better error reporting.